ooniprobe/View/Settings/SettingsCategoriesTableViewController.m

```diff
@@ -46,11 +48,38 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     if ([current isEqualToString:@"about_ooni"]){
         [self performSegueWithIdentifier:current sender:self];
     }
+    else if ([current isEqualToString:@"send_email"]){
+        [self sendEmail];
+    }
     else
         [self performSegueWithIdentifier:@"toSettings" sender:self];
     [tableView deselectRowAtIndexPath:indexPath animated:YES];
 }
 
+-(void)sendEmail{
+    if ([MFMailComposeViewController canSendMail]){
+        MFMailComposeViewController *mailer = [[MFMailComposeViewController alloc] init];
+        mailer.mailComposeDelegate = self;
+        [mailer setSubject:[NSString stringWithFormat:@"[bug-report] OONI Probe iOS %@", [[[NSBundle mainBundle] infoDictionary] valueForKey:@"CFBundleShortVersionString"]]];
+        NSArray *toRecipients = [NSArray arrayWithObjects:@"contact@openobservatory.org", nil];
```

Comment:
We should set the to field to be the `bugs@openobservatory.org` alias

Reply:
So update the ticket. In the instruction we have to use "contact@openobservatory.org"

Reply:
Updated. I had listed the new alias in a comment, but it got a bit buried.

Reply:
Updated